README.md

```diff
@@ -1,7 +1,7 @@
 # gatsby-plugin-drive
 
-Downloads and caches a Google Drive folder that you can then query with `gatsby-source-filesystem`.
-Optionally exports Google Docs to a usable format.
+Downloads and caches a Google Drive folder that you can then query with
+`gatsby-source-filesystem`. Optionally exports Google Docs to a usable format.
```

Comment:
I think the formatting got changed unnecessarily. Could you only change what's important for this feature?